acquia/acquia_cms
# ACMS-1060: Add allowed plugins in Composer

**Motivation**
Fixes #[1060](https://backlog.acquia.com/browse/ACMS-1060)

**Proposed changes**
- Update composer to ^2.2
- add plugins in allowed section

**Alternatives considered**
NA

**Testing steps**
- update composer to ^2.2
- run composer install
- it should not be blocked by any plugins when running composer install

**Merge requirements**
